bitnami/mongodb-sharded/README.md

```diff
@@ -288,11 +288,11 @@ Specify each parameter using the `--set key=value[,key=value]` argument to `helm
 
 ```bash
 $ helm install my-release \
-  --set mongodbRootPassword=secretpassword,mongodbUsername=my-user,mongodbPassword=my-password,mongodbDatabase=my-database \
+  --set shards=4,configsvr.replicas=3,shardsvr.dataNode.replicas=2 \
```

Comment:
```suggestion
  --set  auth.rootPassword=secretPassword,auth.username=my-user,auth.password=my-password,auth.database=my-database \
```

Reply:
I can not find those parameters in the value.yaml.
Is one of use looking into to "wrong" value.yaml?